lib/assessment.js

```diff
@@ -43,7 +47,33 @@ var quad;
 //   the second through last number from the first
 
 /* your code starts here */
-var inject;
+var inject = function(a, b) {
+  var n;
+  
+  if (b === "+" || !b) {
```

Comment:
This is reasonable. It should be noted that this will also trigger the `+` solution if `operation` is an empty string, which may or may not be what we want; however, that's not a test case that the spec described, so no points off.